README.md

```diff
@@ -225,7 +225,7 @@ _(scroll right on table to see all 5 columns)_
 | Uber                           | [github.com (screenshot)](https://github.com/calebnance/expo-uber/blob/master/screenshots/screenshare-3.png?raw=true)                                                  | [GitHub](https://github.com/calebnance/expo-uber)                                                                                          | React native, expo                                                             | ![GitHub Repo stars](https://img.shields.io/github/stars/calebnance/expo-uber)                                                                                                                 |
 | Uber                           | [youtube.com](https://youtu.be/ya1lsXS2Wvw)                                                                                                                            | [GitHub](https://github.com/Santos-Enoque/uber_clone_with_flutter)                                                                         | Flutter                                                                        | ![GitHub Repo stars](https://img.shields.io/github/stars/Santos-Enoque/uber_clone_with_flutter)                                                                                                |
-| Uber Eats                      | [github.com (APK)](https://github.com/sergeyCodenameOne/UberEatsClone/releases/download/v1.0/Grub.apk)                                                                 | [GitHub](https://github.com/sergeyCodenameOne/UberEatsClone)                                                                               | Java, Kotlin                                                                   | ![GitHub Repo stars](https://img.shields.io/github/stars/sergeyCodenameOne/UberEatsClone)                                                                                                      |
+| Uber Eats                     | [enatega.com](https://enatega.com)                                                                 | [GitHub](https://github.com/ninjas-code-official/food-delivery-multivendor)                                                                               | Java, Kotlin                                                                   | ![GitHub Repo stars](https://img.shields.io/github/stars/ninjas-code-official/food-delivery-multivendor)                                                                                                      |
```

Comment:
Hi @SharanGoharKhan, could you also include the tech stack? This new project works with other technologies.

Reply:
Hi @EliasLeguizamon123 changes has been made, technologies has been added